### ISSUE TITLE
Deoplete not working despite no issues in :checkhealth

### ISSUE THREAD

**Issue by pwklein** [Shougo/deoplete.nvim#1120]
**Warning:  I will close the issue without the minimal init.vim and the reproduction instructions.**

# Problems summary
get error message 
`[deoplete] jedi module is not found.  You need to install it..  Use :messages / see above for error details.`
autocompletion then not working

## Expected
no error message and working auto completion
No errors are shown in `:checkhealth`, pynvim is correctly installed, ran `:UpdateRemotePlugins` several times

## Environment Information

 * deoplete version (SHA1): 7535773f9dba82bd8077a2319b7aa8b275dde25a

 * OS: Arch Linux

 * neovim/Vim `:version` output:
:version
```
NVIM v0.4.4
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/bin/cc -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -p
ipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -DMIN_LOG_LEVEL=
3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -
Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-p
rotector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLA
RATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/bu
ild/neovim/src/build/config -I/build/neovim/src/neovim-0.4.4/src -I/usr/include
-I/build/neovim/src/build/src/nvim/auto -I/build/neovim/src/build/include
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

 * `:checkhealth` or `:CheckHealth` result(neovim only):

```
health#deoplete#check
========================================================================
## deoplete.nvim
  - OK: exists("v:t_list") was successful
  - OK: has("timers") was successful
  - OK: has("python3") was successful
  - OK: Require Python 3.6.1+ was successful
  - OK: Require msgpack 1.0.0+ was successful
  - INFO: If you're still having problems, try the following commands:
    - $ export NVIM_PYTHON_LOG_FILE=/tmp/log
    - $ export NVIM_PYTHON_LOG_LEVEL=DEBUG
    - $ nvim
    - $ cat /tmp/log_{PID}
    - and then create an issue on github

health#jedi#check
========================================================================
## jedi
#### Jedi-vim debug information


##### jedi-vim version


 - jedi-vim git version: a17b033
 - jedi git submodule status:  3b7106ae71cb7bd3431101d169a0110ee0c179aa pythonx/jedi (v0.17.2)
 - parso git submodule status:  c864ca60d1a2f36524a4f79013a7de223e65e19d pythonx/parso (v0.6.0)


##### Global Python


Using Python version 3 to access Jedi.
 - global sys.executable: `/usr/bin/python3`
 - global sys.version: `3.8.5 (default, Jul 27 2020, 08:42:51), [GCC 10.1.0]`
 - global site module: `/usr/lib/python3.8/site.py`

##### Jedi

 - path: `/home/pwk/.local/share/nvim/site/autoload/plugged/jedi-vim/pythonx/jedi/jedi/__init__.py`
 - version: 0.17.2

##### Jedi environment: <SameEnvironment: 3.8.5 in /usr>


 - executable: /usr/bin/python3
 - sys_path:
    - `/usr/lib/python38.zip`
    - `/usr/lib/python3.8`
    - `/usr/lib/python3.8/lib-dynload`
    - `/usr/lib/python3.8/site-packages`

##### Known environments


 - <Environment: 3.8.5 in /usr> (/usr/bin/python3.8)

 - <Environment: 2.7.18 in /usr> (/usr/bin/python2.7)



##### Settings



g:jedi#use_splits_not_buffers = 'right' (default: 1)
g:jedi#completions_enabled = 0 (default: 1)


  omnifunc=htmlcomplete#CompleteTags
	Last set from /usr/share/nvim/runtime/ftplugin/html.vim line 24
  completeopt=menuone,longest,preview
	Last set from ~/.local/share/nvim/site/autoload/plugged/jedi-vim/plugin/jedi.vim line 36


health#vimtex#check
========================================================================
## vimtex
  - OK: Vim version should have full support!
  - WARNING: Zathura requires xdotool for forward search!
  - WARNING: Compiler callbacks will not work!
    - ADVICE:
      - `neovim-remote` / `nvr` is required for callbacks to work with neovim
      - Please also set |g:vimtex_compiler_progname| = 'nvr'
      - For more info, see :help |vimtex-faq-neovim|

health#nvim#check
========================================================================
## Configuration
  - OK: no issues found

## Performance
  - OK: Build type: Release

## Remote Plugins
  - OK: Up to date

## terminal
  - INFO: key_backspace (kbs) terminfo entry: key_backspace=\177
  - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~
  - INFO: $COLORTERM='truecolor'

health#provider#check
========================================================================
## Clipboard (optional)
  - OK: Clipboard tool found: xclip

## Python 2 provider (optional)
  - INFO: Using: g:python_host_prog = "/usr/bin/python2"
  - INFO: Executable: /usr/bin/python2
  - INFO: Python version: 2.7.18
  - INFO: pynvim version: 0.4.1
  - OK: Latest pynvim is installed.

## Python 3 provider (optional)
  - INFO: Using: g:python3_host_prog = "/usr/bin/python3"
  - INFO: Executable: /usr/bin/python3
  - INFO: Python version: 3.8.5
  - INFO: pynvim version: 0.4.1
  - OK: Latest pynvim is installed.

## Ruby provider (optional)
  - INFO: Ruby: ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-linux]
  - WARNING: `neovim-ruby-host` not found.
    - ADVICE:
      - Run `gem install neovim` to ensure the neovim RubyGem is installed.
      - Run `gem environment` to ensure the gem bin directory is in $PATH.
      - If you are using rvm/rbenv/chruby, try "rehashing".
      - See :help |g:ruby_host_prog| for non-standard gem installations.

## Node.js provider (optional)
  - WARNING: `node` and `npm` (or `yarn`) must be in $PATH.
    - ADVICE:
      - Install Node.js and verify that `node` and `npm` (or `yarn`) commands work.

```

## Provide a minimal init.vim/vimrc with less than 50 lines (Required!)

```


let g:python_host_prog = expand('/usr/bin/python2')
let g:python3_host_prog = expand('/usr/bin/python3')

let g:deoplete#enable_at_startup = 1
autocmd InsertLeave,CompleteDone * if pumvisible() == 0 | pclose | endif
inoremap <expr><tab> pumvisible() ? "\<c-n>" : "\<tab>"

" disable autocompletion, cause we use deoplete for completion
let g:jedi#completions_enabled = 0

" open the go-to function in split, not another buffer
let g:jedi#use_splits_not_buffers = "right"

call plug#begin('~/.local/share/nvim/site/autoload/plugged')
Plug 'machakann/vim-highlightedyank'
Plug 'chrisbra/Colorizer'
Plug 'cjrh/vim-conda'
Plug 'scrooloose/NERDTree'
Plug 'sheerun/vim-polyglot'
Plug 'jiangmiao/auto-pairs'
Plug 'vim-airline/vim-airline'
Plug 'vim-airline/vim-airline-themes'
Plug 'Shougo/deoplete.nvim', { 'do': ':UpdateRemotePlugins' }
Plug 'tpope/vim-fugitive'
Plug 'zchee/deoplete-jedi'
Plug 'scrooloose/nerdcommenter'
Plug 'sbdchd/neoformat'
Plug 'davidhalter/jedi-vim'
Plug 'lervag/vimtex'
call plug#end()

" This is new style
call deoplete#custom#var('omni', 'input_patterns', {
      \ 'tex': g:vimtex#re#deoplete
      \})
let g:vimtex_view_method = 'zathura'


```

## How to reproduce the problem from neovim/Vim startup (Required!)

 1. Install python 2 and 3 from the official arch repos 
 2. Install pynvim for both of them (for python 3 from the official repo, for python 2 from aur)
 3. Install deoplete with plugged
 4.  open a python script

## Generate a logfile if appropriate

 1. export NVIM_PYTHON_LOG_FILE=/tmp/log
 2. export NVIM_PYTHON_LOG_LEVEL=DEBUG
 3. nvim -u minimal.vimrc
 4. some works
 5. cat /tmp/log_{PID}

[log_py3_script.txt](https://github.com/Shougo/deoplete.nvim/files/5047657/log_py3_script.txt)
[log_py3_rplugin.txt](https://github.com/Shougo/deoplete.nvim/files/5047659/log_py3_rplugin.txt)

**Comment by patmessina** [comment]
Did you pip run `pip3 install --user jedi` from https://github.com/deoplete-plugins/deoplete-jedi?

**Comment by Shougo** [comment]
Yes.   You need to install jedi module for deoplete-jedi.

**Comment by Shougo** [comment]
You should read deoplete-jedi documentation.

**Comment by pwklein** [comment]
For reference:

I did not install the python jedi module via pip since I am on Arch and I do not want to install packages globally using pip as this will mess up my system. I installed the packaged version python-jedi via pacman, which is currently v 0.17.1. This version however contains a bug that has been fixed in version 0.17.2
So the bug was on the jedi side and the Arch python-jedi package needs to be updated.
